applications/openshift/general/kube_descheduler_operator_exists/rule.yml

```diff
@@ -0,0 +1,42 @@
+documentation_complete: true
+
+title: Ensure that the Kube Descheduler operator is deployed
+
+description: |-
+    If there is an increased risk of external influences and a very high need for protection, pods should be stopped and restarted regularly.
+    No pod should run for more than 24 hours. The availability of the applications in the pod should be ensured.
+
+rationale: |-
+    If there is an increased risk of external influences and a very high need for protection, pods should be stopped and restarted regularly.
+
+identifiers: {}
+
+references:
+  bsi: APP.4.4.A21
+
+severity: medium
+
+ocil_clause: "No CRDs from the Kube Descheduler operator were found"
+
+ocil: |-
+    Run the following command to retrieve the customresourcedefinitions objects in the system:
+    <pre>$ oc get crds </pre>
+    Make sure there is a crd for the Kube Descheduler operator.
+
+{{% set jqfilter = '[.items[] | select(.metadata.name =="kubedeschedulers.operator.openshift.io") | .metadata.name]' %}}
+
+warnings:
+- general: |-
+    {{{ openshift_filtered_cluster_setting({'/apis/apiextensions.k8s.io/v1/customresourcedefinitions?limit=500': jqfilter}) | indent(4) }}}
+
+template:
+  name: yamlfile_value
+  vars:
+    ocp_data: "true"
+    filepath: {{{ openshift_filtered_path('/apis/apiextensions.k8s.io/v1/customresourcedefinitions?limit=500', jqfilter) }}}
```

Comment:
By the way, would it be more reliable and simpler to check for the operator's subscription instead of looking for its CRDs?
Check what [container_security_operator_exists](https://github.com/ComplianceAsCode/content/blob/cd0c0455895e586a7f6c4a8720864cd887cb9634/applications/openshift/risk-assessment/container_security_operator_exists/rule.yml#L55) rule does.
https://github.com/ComplianceAsCode/content/blob/cd0c0455895e586a7f6c4a8720864cd887cb9634/applications/openshift/risk-assessment/container_security_operator_exists/rule.yml#L55